sycl/include/sycl/ext/oneapi/matrix/matrix-unified.hpp

```diff
@@ -99,7 +88,7 @@ class wi_data {
     return jm.cuda_impl.wi_marray.size();
 #else
     throw runtime_error("get_wi_data is available using: "
-                        "ext::intel::experimental::matrix::get_wi_data.",
+                        "ext::oneapi::detail::get_wi_data.",
                         PI_ERROR_INVALID_DEVICE);
```

Comment:
We should not advise users to use this. When does this runtime error occur?

Reply:
ok, i see. wi_data class here is in sycl::ext::oneapi::experimental::matrix namespace and it is for NV. the errmsg is for intel users who uses NV's wi_data

Reply:
how about 
get_wi_data is available using: ext::oneapi::detail::get_wi_data but intel users are expected to use joint_matrix_copy instead

Reply:
We should never advise users to use anything from detail namespace. Detail namespace are implementation details and can change at any time. It is not part of documented API.